pkg_ci.Rmd

```diff
@@ -6,9 +6,20 @@ This chapter summarizes our guidelines about continuous integration after explai
 Along with [last chapter](#building), it forms our guidelines for Software Peer Review.
 ```
 
+## What is continuous integration (CI)?
+
+Continuous integration automatically runs tests on software.
+In the case of rOpenSci, CI practically means that a set of tests will be automatically run via github, every time that you make a commit or pull request to github.
+
+For CI to be meaningful you have to have tests set up that work locally using `R CMD check`, see [testing](/building.html#testing).
```

Comment:
Can this maybe be rephrased to a more pragmatic word than "meaningful"? Suggestion only: Something along the lines of "CI helps automate general package checks such as `R CMD check`. One of the main uses is testing ..."

Reply:
Changed to:
> CI automates the running of general package checks such as `R CMD check`, see [testing](/building.html#testing).